agent/src/vscode-shim.ts

```diff
@@ -945,12 +942,13 @@ const _commands: Partial<typeof vscode.commands> = {
 _commands?.registerCommand?.('workbench.action.reloadWindow', () => {
     // Do nothing
 })
-_commands?.registerCommand?.('setContext', (key, value) => {
+_commands?.registerCommand?.('setContext', (...args) => {
+    const [key, value] = args
```

Comment:
Does this not work if you just leave it as
```suggestion
_commands?.registerCommand?.('setContext', (key, value) => {
```

Reply:
Yup that works, thank you!
![image](https://github.com/user-attachments/assets/96bfef60-16ae-427f-95f3-d12a3afdf79a)